app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -104,9 +104,9 @@ def handle_remember_device
     save_remember_device_preference
   end
 
-  def handle_valid_otp_for_context
+  def handle_valid_otp_for_context(auth_method)
```

Comment:
Noting the inconsistency between this method using a positional argument and `handle_valid_otp_for_authentication_context` below using a keyword argument. Should they be consistent?